sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpUtils.java

```diff
@@ -154,6 +154,17 @@ public static String hideTickets(@NonNull String tickets) {
         return TOKEN_VALUE_PATTERN.matcher(tickets).replaceAll(":***");
     }
 
+    /**
+     * Hide JWT token value in Authorization header string.
+     *
+     * @param token string header value.
+     * @return obfuscated token string header value.
+     */
+    public static String hideAuthToken(@NonNull String token) {
+        String prefix = token.split("\\s+")[0];
```

Comment:
You can just print "***" instead of "Bearer ***" but not a big deal

Reply:
I prefer "Bearer ***"